packages/gasket-fetch/lib/index.d.ts

```diff
@@ -0,0 +1,4 @@
+async function fetch(input: RequestInfo, init?: RequestInit): Promise<Response>;
```

Comment:
Getting an error on this line: `'async' modifier cannot be used in an ambient context.`. Is this an important error?

Reply:
Removing `async` should work fine; `async` is only needed for implementations, not type declarations.